src/main/java/gate/util/Files.java

```diff
@@ -177,7 +177,7 @@ public static String getGateResourceAsString(String resourceName)
     if (resourceStream == null)
       throw new IOException("No such resource on classpath: " + resourceName);
     try {
-      return IOUtils.toString(resourceStream);
+      return IOUtils.toString(resourceStream,Charset.defaultCharset().name());
```

Comment:
This depends on os settings ... Would be better if it was UTF-8 I think.

Reply:
Ok